services/user.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.validators import validate_email
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+def email_validator(email: str) -> None:
```

Comment:
As I mentioned earlier, validation would be more appropriate inside the model. You can redefine the fields for User and overwrite clean() method there